nesc/whip6/apps/AsyncWriteDemo/README

```diff
@@ -0,0 +1,28 @@
+This application serves the purpose of verifing whether uart is able to write a single byte asynchronously.
+It runs in two phases.
+
+In the first phase uart writes consecutive alphabet characters, from 'a' to 'z', one by one, each byte in a separate timer-callback function.
+In the second phase, alphabet characters (from 'a' to 'z') are written within a loop, each byte in a separate iteration.
+
+*************
+How to use it
+*************
+1. Install the application.
+2. Start minicom (make sure you have connected minicom correctly to the device).
+3. Reset the board and observe both: the output printed in the minicom terminal and leds.
```

Comment:
You may want to add a note, that the application is one shot. In order to test it again, one needs to reset the board again.